config/cmake/jrunTest.cmake

```diff
@@ -131,7 +131,7 @@ if (NOT TEST_SKIP_COMPARE)
     file (READ ${TEST_FOLDER}/${TEST_REFERENCE} TEST_STREAM)
     list (LENGTH TEST_STREAM test_len)
     if (test_len GREATER 0)
-      if (WIN32 OR MINGW)
+      if (WIN32)
```

Comment:
This and the next are workarounds for tests on a windows host and might not be valid for a linux host.

Reply:
if the issue is crlf, you will get the with wine, so this is fine